### DISCUSSION TITLE
Option to disable Netherite armor's knockback resistance

### DISCUSSION THREAD

**Discussion by OmegaShaman** [PurpurMC/Purpur#988] [Feature Requests]
Hello, so recently I've been having an issue on my server in which people are begging me to remove Netherite armor's knockback resistance. The problem is I can't change that and it is also inconsistent when it comes to people fighting. People keep complaining about how people can't have fun fights anymore because it's too tanky, and this one plugin I have does a horrible job at nullifying it.

Is it possible to add a config option that allows someone to disable the knockback resistance on Netherite armor, as well as a way to turn of the inconsistencies?

(example:
```
netherite-armor:
  remove-knockback-resistance: true
  consistent-knockback: true
```
)

If its possible to toggle Netherite's fire resistance it's certainly possible to turn off knockback resistance on armor.

**Comment by YouHaveTrouble** [comment]
You can always get plugin to modify the armor and knockback resistance of netherrite armor instead of trying to overwrite the knockback itself ;P

**Reply by SageSphinx63920** [reply]
But it would be nicer if purpur has an option to change the knockback of armour in general

**Comment by OmegaShaman** [comment]
> You can always get plugin to modify the armor and knockback resistance of netherrite armor instead of trying to overwrite the knockback itself ;P

Wish I could, but as I said in the request, the plugin I have is terrible and doesn't work in laggy conditions well, and even in normal conditions it has a hard time negating Netherite knockback resistance.

**Reply by YouHaveTrouble** [reply]
I meant attributes on the armor. then the server does all the work involving knockback for you.

**Reply by OmegaShaman** [reply]
I can't find a plugin that changes Knockback attributes on Armor, so I honestly think it's just better if it was built into purpur because then it would affect all NBT data (I think).
